src/styles/blocks/card.scss

```diff
@@ -0,0 +1,116 @@
+$color_white: #fff;
+$color_black: #060b35;
+$color_blue: #00acdc;
+$color_grey: #616070;
+$radius: 5px;
+$button_width: 166px;
+$button_height: 40px;
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  flex-shrink: 0;
+  border-radius: $radius;
+  border: 1px solid #f3f3f3;
+  background: $color_white;
+  margin: auto;
+  padding: 32px 16px 16px;
+
+  &__foto {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__text {
+    color: $color_black;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
```

Comment:
it's better to use just `margin-bottom` and reset default margin for `p` and `h2` similar you did with `body`